star/job/run_star_support.f90

```diff
@@ -3822,7 +3826,69 @@ subroutine add_fpe_checks(id, s, ierr)
          end if
 
       end subroutine add_fpe_checks
-
+      
+      
+      
+      subroutine multiply_tolerances(id, s, ierr)
+         integer, intent(in) :: id
+         type (star_info), pointer :: s
+         integer, intent(out) :: ierr
+         integer :: status
+         
+         real(dp) :: mesh_delta_coeff_factor = 1
+         real(dp) :: time_delta_coeff_factor = 1
+         real(dp) :: max_model_number_factor = 1
+         character(len=20) :: mesh_delta_coeff_factor_str
+         character(len=20) :: time_delta_coeff_factor_str
+         character(len=20) :: max_model_number_factor_str
+         
+         include 'formats'
+         
+         ierr = 0
+         call GET_ENVIRONMENT_VARIABLE('MESA_MESH_DELTA_COEFF_FACTOR', mesh_delta_coeff_factor_str, STATUS=status)
```

Comment:
We should be checking if status /=0

Reply:
OK, I've implemented it! [I had thought it might have been fine for people to specify one of the env vars without caring about the others (so for example they could increase mesh_delta without caring about the others), so I had skipped it.]